neurom/features/neurite_features.py

```diff
@@ -97,3 +116,53 @@ def section_path_distances(neurites, use_start_point=False, neurite_type=TreeTyp
     magic_iter = (_sec.start_point_path_length if use_start_point
                   else _sec.end_point_path_length)
     return iter_neurites(neurites, magic_iter, _ttc(neurite_type))
+
+
+@ensure_neurite_list
+def section_radial_distances(neurites, origin=None, use_start_point=False,
```

Comment:
I don't like it either, but I can't think of a better way to do it at the moment.